packages/tools/pluggable-widgets-tools/src/typings-generator/generateClientTypes.ts

```diff
@@ -1,7 +1,12 @@
 import { Property } from "./WidgetXml";
 import { capitalizeFirstLetter, extractProperties } from "./helpers";
 
-export function generateClientTypes(widgetName: string, properties: Property[], isNative: boolean): string[] {
+export function generateClientTypes(
```

Comment:
Could it be better to pass `SystemProperties` next to `Properties` in this api iso `isLabeled` flag?

Reply:
Indeed, better for symmetry